src/test/java/de/tum/in/www1/artemis/metis/PostIntegrationTest.java

```diff
@@ -167,6 +176,19 @@ public void testCreateCourseWidePost() throws Exception {
         verify(groupNotificationService, times(1)).notifyAllGroupsAboutNewCoursePost(createdPost, course);
     }
 
+    @Test
+    @WithMockUser(username = "student1", roles = "USER")
+    public void testCreateConversationPost() throws Exception {
```

Comment:
This is the only test about creating posts in a conversation.
I think it would be important to have some additional tests for this. For example, testing the creation of a post when you are not part of the conversation would be important imo.

Reply:
Will be done in the follow-up PR.